ftpslib/ftps_zephir_dir

```diff
@@ -1,11 +1,6 @@
-#!/bin/sh 
+#!/bin/bash 
```

Comment:
Debian `/bin/sh` is dash, which sometimes leads to surprises. Better to specify bash.